src/link/subscriptions/index.ts

```diff
@@ -47,6 +47,29 @@ function isLikeErrorEvent(err: unknown): err is Event {
   return isNonNullObject(err) && err.target?.readyState === WebSocket.CLOSED;
 }
 
+/**
+ * The `GraphQLWsLink` is a terminating link sends GraphQL operations over a
+ * WebSocket connection using the [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) library. It's used most
+ * commonly with GraphQL [subscriptions](https://apollographql.com/docs/react/data/subscriptions/),
+ *
+ * > [!NOTE]
+ * > This link works with the newer `graphql-ws` library. If your server uses
+ * > the older `subscriptions-transport-ws` library, you should use the
```

Comment:
```suggestion
 * > This link works with the `graphql-ws` library. If your server uses
 * > the deprecated `subscriptions-transport-ws` library, you should use the
```

I'd suggest we change the wording here. Should we also deprecate `WebSocketLink`? The host library is deprecated for four years now.

Reply:
Good call. Updated in 892383ce432e3a71d8a0820f9a2729dbdbf3117e